pkgs/applications/graphics/gimp/default.nix

```diff
@@ -116,9 +113,6 @@ in stdenv.mkDerivation rec {
     shared-mime-info
     libwebp
     libheif
-    python
-    # Duplicated here because python.withPackages does not expose the dev output with pkg-config files
-    python2.pkgs.pygtk
```

Comment:
Lets move this behind a flag which is off by default

Reply:
Sounds good, will make the change later.

Reply:
Changed to a conditional